pkg/machine/config.go

```diff
@@ -14,16 +14,17 @@ import (
 )
 
 type InitOptions struct {
-	CPUS         uint64
-	DiskSize     uint64
-	IgnitionPath string
-	ImagePath    string
-	IsDefault    bool
-	Memory       uint64
-	Name         string
-	TimeZone     string
-	URI          url.URL
-	Username     string
+	CPUS                  uint64
+	DiskSize              uint64
+	IgnitionPath          string
+	IgnitionOverridePaths []string
```

Comment:
To keep my OCD happy, please flip these lines, 'IgnitionO*, should could before "IgnitionP*"